code/script.js

```diff
@@ -12,36 +54,308 @@ const showMessage = (message, sender) => {
       <section class="user-msg">
         <div class="bubble user-bubble">
           <p>${message}</p>
-        </div>
-        <img src="assets/user.png" alt="User" />  
+        </div> 
       </section>
     `
   } else if (sender === 'bot') {
     chat.innerHTML += `
       <section class="bot-msg">
-        <img src="assets/bot.png" alt="Bot" />
+        <img src="./assets/chatbot.png" alt="Bot" />
         <div class="bubble bot-bubble">
           <p>${message}</p>
         </div>
       </section>
-    `
+   `
   }
+
   // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
   chat.scrollTop = chat.scrollHeight
 }
 
-// Starts here
-const greeting = () => {
-  showMessage(`Hello there, What's your name?`, 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+//input from user
+const handleResponse = (response) => {
+  showMessage(response, 'user');
+}
+
+// Bot chat messages
+// .0
+const nights = (fName) => {
+  questionNumber = 1;
+  showMessage(`Hi ${fName}, how many nights do you want to book?`, 'bot');
+}
+
+// .1
+const persons = (nights) => {
+  if (Number(nights) > 0 && Number(nights) <= 28) {
+    showMessage(`How many are going to travel?`, 'bot');
+    questionNumber++
+    bookingSentence += `We're booking a ${nights} day/s trip`
+  } else {
+    showMessage(`That is not a valid number, try again.`, 'bot');
+  }
+}
+
+// .2 with buttons instead of text input
+const where = (persons) => {
+  if (Number(persons) > 0 && Number(persons) <= 14) {
```

Comment:
Again your users need to be informed about the max number of friends they can take with them 😃